app/code/community/Reverb/ProcessQueue/Helper/Task.php

```diff
@@ -0,0 +1,28 @@
+<?php
+/**
+ * Author: Sean Dunagan (https://github.com/dunagan5887)
+ * Created: 12/9/15
+ */
+
+class Reverb_ProcessQueue_Helper_Task extends Mage_Core_Helper_Data
+{
+    const STALE_TIME_LAPSE = '-2 weeks';
+
+    /**
+     * The calling block is expected to catch exceptions
+     *
+     * @param null|string $task_code
+     * @return int - Number of rows deleted
+     */
+    public function deleteStaleSuccessfulTasks($task_code = null)
+    {
+        $current_gmt_timestamp = Mage::getSingleton('core/date')->gmtTimestamp();
+        $stale_timestamp = strtotime(self::STALE_TIME_LAPSE, $current_gmt_timestamp);
+        $stale_date = date('Y-m-d H:i:s', $stale_timestamp);
+
+        $rows_deleted = Mage::getResourceSingleton('reverb_process_queue/task')
+                            ->deleteSuccessfulTasks($task_code, $stale_date);
```

Comment:
The cron executing this task is only set to run nightly at 4:36AM. I could add an index for last_executed_at on the table to optimize performance, but the creation of the index on tables with 10,000+ rows wouldn't be a quick operation. If you're concerned I can populate my local instance table with 15,000 rows and test execution. But again, this will only be running in the middle of the night